data/io.elementary.files.appdata.xml.in.in

```diff
@@ -58,6 +58,29 @@
     <content_attribute id="money-gambling">none</content_attribute>
   </content_rating>
   <releases>
+    <release version="Next" date="Future" urgency="medium">
+      <description>
+        <p>New features:</p>
+        <ul>
+          <li>Initial git plugin support</li>
+          <li>Check hicolor theme if no elementary icon before falling back to default</li>
+          <li>Follow global history setting when restoring and saving tabs</li>
+        </ul>
+        <p>Other updates:</p>
+        <ul>
+          <li>Asynchronous undo/redo</li>
+          <li>Ensure tabs showing same folder share an Async object</li>
+          <li>Fix drag with secondary button on empty space</li>
+          <li>Improve appearance of diskspace indicator</li>
+          <li>Improve pathbar animation</li>
```

Comment:
I think these were in the last release weren't they?

Reply:
I was referring to

https://github.com/elementary/files/commit/4f00d7b99d62f07e93d7d06f0da6823d1b730a9a
https://github.com/elementary/files/commit/5fe5709e6253680a44abd7284c3c0d416da551cc